README.md

```diff
@@ -48,7 +48,7 @@ Reference the Bootstrap CSS stylesheet in `public/index.html` (optional):
 
 In the app directory, generate the files for the resource you want:
 
-    $ api-platform-generate-crud http://localhost src/ --resource foo
+    $ api-platform-generate-crud https://demo.api-platform.com/index.jsonld src/ --resource foo
```

Comment:
Generator doesn't need index.jsonld specified
I had many problems with generating endpoints by this generator but it was always the problem with backend: configuration, cache etc.

If I may suggest, instead writing it to readme file, create new Troubleshooting.md and all this issues/problems can land there.
I can add my experience with custom endpoints with data providers and generator.